web/webkit/src/main/scala/net/liftweb/http/LiftSession.scala

```diff
@@ -2345,15 +2345,31 @@ class LiftSession(private[http] val _contextPath: String, val underlyingId: Stri
    * the CometActor is instantiated.  If the CometActor already exists
    * in the session, the message will be sent immediately.  If the CometActor
    * is not yet instantiated, the message will be sent to the CometActor
-   * as part of setup (@see queueCometMessage) regardless of if it is created as part
+   * as part of setup (see queueCometMessage) if it is created as part
+   * of the current HTTP request/response cycle.
+   *
+   * @param msg the message to send to the CometActor
+   */
+  def sendCometMessage[T](msg: Any)(implicit cometManifest: Manifest[T]): Unit = {
+    val castClass = cometManifest.runtimeClass.asInstanceOf[Class[T]]
```

Comment:
This is probably a bit silly, but this is the way the other typed method did it - and I weighed the likelihood that this was a mistake as lower than the likelihood that there was some magic that made this useful that I just didn't know about. Hah.

Could this provide some assistance to dealing with erasure per chance?

I'm actually scratching my head at this now too. Maybe it was just a fluke that we should fix. heh.